setup.cfg

```diff
@@ -25,16 +26,15 @@ packages = find:
 python_requires = >=3.7
 zip_safe = False
 install_requires =
-    Faker>=2.0.3
-    invenio-rdm-records>=1.3.6,<2.0.0
+    invenio-rdm-records>=10.0.0,<11.0.0
 
 [options.extras_require]
 tests =
     pytest-black>=0.3.0
     importlib-metadata<5.0.0
-    invenio-app>=1.3.4,<2.0.0
-    invenio-db[postgresql,mysql]>=1.0.14,<2.0.0
-    invenio-rdm-records>=1.2.0,<2
+    invenio-app>=1.4.0,<2.0.0
+    invenio-db[postgresql,mysql]>=1.1.0,<2.0.0
+    invenio-rdm-records>=10.0.0,<11.0.0
```

Comment:
maybe it would simpler to require `invenio-app-rdm>=12.0.0rc2,<13` instead of the modules individually.
the configuration in the README also references invenio-app-rdm.